terraform/dev/api.tf

```diff
@@ -21,6 +21,49 @@ resource "aws_lambda_function" "api" {
   }
 }
 
+resource "aws_lambda_function" "etsToSbc" {
+  description      = "etsToSbc function ${local.namespace}"
+  function_name    = local.ets_to_sbc
+  role             = aws_iam_role.lambda.arn
+  runtime          = "nodejs14.x"
+  filename         = var.ets_to_sbc_zip
+  source_code_hash = filebase64sha256(var.ets_to_sbc_zip)
+  handler          = "lambda.handler"
+  memory_size      = 1024
+  timeout          = 10
+
+  environment {
+    variables = {
+      DB_SERVER   = "dummy"
+      DB_PORT     = "dummy"
+      DB_USER     = "dummy"
+      DB_PASSWORD = "dummy"
+      DB_NAME     = "dummy"
+      VERSION     = var.git_version
+    }
+  }
+}
+
+resource "aws_lambda_permission" "allow_cloudwatch_to_call_etsToSbc" {
```

Comment:
File not formatted. https://marketplace.visualstudio.com/items?itemName=HashiCorp.terraform